learning/tour-of-beam/learning-content/java/introduction/introduction-concepts/basic-concepts/description.md

```diff
@@ -0,0 +1,134 @@
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+# Tour of Beam Programming Guide
```

Comment:
Alex, we discussed this with Abzal and we propose to have pipeline options and custom options in a single unit since there is no value in having two examples for those topics. The concepts of options and custom options are similar and could be illustrated with a single example

Reply:
ok